src/ASM/ASMs2D.h

```diff
@@ -529,6 +529,7 @@ class ASMs2D : public ASMstruct, public ASM2D
   //! \param[out] XC Coordinates of the element corners
   void getElementCorners(int i1, int i2, std::vector<Vec3>& XC) const;
 
+  using ASMbase::generateThreadGroups;
   //! \brief Generates element groups for multi-threading of interior integrals.
```

Comment:
why is it only ASMs2D that require this using-statement? The other 2D classes (mx and Lag) also reimplements only one of the generateThreadGroups methods, so if this one gives warning and not the other two it does not make sense in my head.